dragonfly/implementations/daq_run_interface.py

```diff
@@ -101,6 +112,9 @@ def start_run(self, run_name):
         '''
         '''
         self.run_name = run_name
+        if self._run_name is None:
+            logger.critical('run_name is not present; all internal run procedures will be squashed')
+            raise core.exceptions.DriplineValueError('<{}> instance <{}> requires a value for "{}" to initialize run and procedures'.format(self.__class__.__name__, self.name, '_run_name'))
```

Comment:
About the logger critical, my thought was that we would want to communicate the failure of this run to the user and slack. But I agree that it doesn't fit well. An exception will be raised either when the run_name insert and return fails. Is the exception message good enough then?